### ISSUE TITLE
Can't install the apk on Android 14

### ISSUE THREAD

**Issue by oefi** [Ohmnibus/quick-dice-roller#11]
I have been enjoying this application for many years. Sadly it can't be installed on Android 14 anymore.

**Comment by DirkPitt1** [comment]
Yeah. It's probably no use, but please, just update the app to work on Android 16. I tried many other dice roll apps, but this is the most convenient option and the easiest to add some custom dice.
